nixos/modules/virtualisation/google-compute-image.nix

```diff
@@ -44,10 +44,18 @@ in
         GZIP compression level of the resulting disk image (1-9).
       '';
     };
+    virtualisation.googleComputeImage.efi =
+      mkEnableOption (lib.mdDoc "Whether to use EFI booting");
```

Comment:
This already adds a `Whether to enable...`:
https://github.com/nixos/nixpkgs/blob/master/lib/options.nix#L103-L103

So just putting something `EFI booting` is enough.